src/loaders/LoaderUtils.js

```diff
@@ -4,11 +4,15 @@
 
 var LoaderUtils = {
 
+	textDecoder: null,
```

Comment:
I think I would put this in a closure inside `decodeText` for now.

Reply:
Ok, done 👍